tests/simple_standard_scenarios_datasets.py

```diff
@@ -0,0 +1,316 @@
+"""Contains functions to read and write simple-standard-scenarios datasets."""
+
+from collections import defaultdict, namedtuple
+from pathlib import Path
+
+import pyspark.sql.functions as F
+from pyspark.sql.types import IntegerType
+from pyspark.sql import SparkSession
+
+from dsgrid.config.mapping_tables import MappingTableRecordModel
+from dsgrid.registry.registry_database import DatabaseConnection, RegistryDatabase
+from dsgrid.registry.registry_manager import RegistryManager
+from dsgrid.utils.files import dump_data, load_data
+from dsgrid.utils.spark import models_to_dataframe
+from dsgrid.utils.utilities import convert_record_dicts_to_classes
+from dsgrid.tests.utils import read_csv_single_table_format, read_parquet_two_table_format
+
+
+REGISTRY_PATH = Path("dsgrid-test-data/filtered_registries/simple_standard_scenarios")
+EXPECTED_DATASET_PATH = REGISTRY_PATH / "expected_datasets"
+BUILDING_COUNTY_MAPPING = {
+    "06037": "G0600370",
+    "06073": "G0600730",
+    "36047": "G3600470",
+    "36081": "G3600810",
+}
+STATS_FILENAME = "raw_stats.json"
+Datasets = namedtuple("Datasets", ["comstock", "resstock", "tempo"])
+
+
+def build_expected_datasets():
+    """Build the expected datasets and save summarized stats to a JSON file."""
+    path = REGISTRY_PATH
+    aeo_com = map_aeo_com_subsectors(
+        map_aeo_com_county_to_comstock_county(
```

Comment:
Should this file somehow be excluded from codecov, since it's purpose is more to test rather than to be tested?

Reply:
I’ll add a codecov.yml